src/const/tab.js

```diff
@@ -0,0 +1,23 @@
+import {withStyles} from "@material-ui/core/styles";
+import Tabs from "@material-ui/core/Tabs";
+import Tab from "@material-ui/core/Tab";
+import React from "react";
+
+export const StyledTabs = withStyles((THEME) => ({
```

Comment:
Babe `StyledTabs` is only used in `src/components/Timer/index.jsx` so you can put it there instead.